modules/sdk-core/src/bitgo/keychain/iKeychains.ts

```diff
@@ -105,6 +113,7 @@ export interface AddKeychainOptions {
   isDistributedCustody?: boolean;
   // indicates if the key is MPCv2 or not
   isMPCv2?: boolean;
+  coinSpecific?: { [coinName: string]: AddLightningKeychainCoinSpecific };
```

Comment:
IMO this is accurate - the caller should check the type

otherwise this will end up a a supertype with all coins adding their CoinSpecific
 
```suggestion
  coinSpecific?: { [coinName: string]: unknown };
```